nekoyume/Assets/_Scripts/Extensions/AvatarStateExtensions.cs

```diff
@@ -16,9 +16,11 @@ public static ArenaAvatarState ToArenaAvatarState(this AvatarState avatarState)
         {
             var arenaAvatarState = new ArenaAvatarState(avatarState);
             arenaAvatarState.UpdateCostumes(avatarState.inventory.Costumes
+                .Where(costume => costume.Equipped)
                 .Select(e => e.NonFungibleId)
                 .ToList());
             arenaAvatarState.UpdateEquipment(avatarState.inventory.Equipments
+                .Where(equipment => equipment.Equipped)
```

Comment:
Could you explain why this `Where` operator added?

Reply:
This is to fix a bug that indicates that all items in the inventory are equipped.